app/views/layouts/component_preview.html.erb

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Component Preview</title>
+
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+    <%= favicon_link_tag "govuk-frontend/govuk/assets/images/favicon.ico" %>
+    <%= favicon_link_tag "govuk-frontend/govuk/assets/images/govuk-mask-icon.svg", rel: "mask-icon", type: "image/svg", color: "#0b0c0c" %>
+    <%= favicon_link_tag "govuk-frontend/govuk/assets/images/govuk-apple-touch-icon.png", rel: "apple-touch-icon", type: "image/png" %>
+    <%= favicon_link_tag "govuk-frontend/govuk/assets/images/govuk-apple-touch-icon-152x152.png", rel: "apple-touch-icon", type: "image/png", size: "152x152" %>
+    <%= favicon_link_tag "govuk-frontend/govuk/assets/images/govuk-apple-touch-icon-167x167.png", rel: "apple-touch-icon", type: "image/png", size: "167x167" %>
+    <%= favicon_link_tag "govuk-frontend/govuk/assets/images/govuk-apple-touch-icon-180x180.png", rel: "apple-touch-icon", type: "image/png", size: "180x180" %>
+    <%= stylesheet_link_tag "accessible-autocomplete/dist/accessible-autocomplete.min" %>
+    <%= stylesheet_link_tag "find_application" %>
+    <%= stylesheet_link_tag "publish_application" %>
+    <%= javascript_include_tag "find/application", "data-turbo-track": "reload", defer: true %>
+    <%= javascript_include_tag "publish/application", "data-turbo-track": "reload", defer: true %>
```

Comment:
having both will probably cause issues later down the line

Reply:
Same response as above